packages/grafana-data/src/types/dataLink.ts

```diff
@@ -1,12 +1,12 @@
-import { ScopedVars } from './ScopedVars';
 import { DataQuery } from './datasource';
+import { InterpolateFunction } from './panel';
 
 /**
  * Callback info for DataLink click events
  */
 export interface DataLinkClickEvent<T = any> {
   origin: T;
-  scopedVars?: ScopedVars;
+  replaceVariables: InterpolateFunction | undefined;
```

Comment:
nit: Out of curiosity why not `replaceVariables?` instead of adding  `| undefined`